tests/core/test_exp_manager.py

```diff
@@ -185,7 +185,7 @@ def test_trainer_loggers(self, tmp_path):
                 "create_checkpoint_callback": False,
                 "exp_dir": str(tmp_path),
                 "create_wandb_logger": True,
-                "wandb_logger_kwargs": {"name": "", "project": ""},
+                "wandb_logger_kwargs": {"name": "", "project": "", "offline": True},
```

Comment:
Does this mean every exp_manager config needs to set this to True to log stuff to your wandb account?